### DISCUSSION TITLE
[V3] Can't test components that use magic render

### DISCUSSION THREAD

**Discussion by edalzell** [livewire/livewire#5944] [7. Bugs (archived)]
I have a full page component, `Member`, that uses the magic render:

![CleanShot 2023-07-24 at 20 36 03@2x](https://github.com/livewire/livewire/assets/6069653/6c8510a5-2a25-4f82-bf0a-c42bb27c5a35)

![CleanShot 2023-07-24 at 20 36 58@2x](https://github.com/livewire/livewire/assets/6069653/2e3f4c2b-9a8f-4a20-9df1-c92795ecb0fa)

<img width="468" alt="CleanShot 2023-07-24 at 20 36 40@2x" src="https://github.com/livewire/livewire/assets/6069653/7e2c484e-7d09-4fab-9c29-c16226b9b088">

This component will render in the browser with no problems, but when I try to test it:
<img width="424" alt="CleanShot 2023-07-24 at 20 37 40@2x" src="https://github.com/livewire/livewire/assets/6069653/9e524867-ddd1-4186-833e-69e6b62a3708">

I get: 
<img width="1275" alt="image" src="https://github.com/livewire/livewire/assets/6069653/719ca691-8ac2-4811-8a0b-23514ff34e9d">

**Comment by joshhanley** [comment]
Have created a failing test PR #5947.
